portal/component-tests/utilisation-report-service/utilisation-report-upload.component-test.js

```diff
@@ -113,7 +114,8 @@ describe(page, () => {
     });
 
     it('should state which report the page is expecting to be uploaded', () => {
-      wrapper.expectText('[data-cy="upload-report-text"]').toRead('Upload January 2023 report');
+      wrapper.expectText('[data-cy="upload-report-text"]').toMatch(/(Upload January 2023 report)/);
```

Comment:
Out of curiosity do we need the `(` and `)`? I've not checked out this branch to check how `expect` works in jest, but regex wise, it seems we could lose it.